### ISSUE TITLE
Add GreenWall Model

### ISSUE THREAD

**Issue by JanineBardey** [RWTH-EBC/AixLib#1143]
**Context:** 

Active green wall are included more and more often into buildings to enhance the indoor air quality and having an effect on humidity, VOC and CO2 level. 

**What is the problem?**

There is no Model included in the AixLib jet, addressing the current trend of building greening. Therefore the interactions of green walls and other systems like AHU cannot be simulated easily. Also no evaluation on the resource consumption of water and energy can be done.

**How do we want to solve it?**

Adding a modelt o the AixLib representing a plant wall including effects of humidification and filtering of CO2,VOC and particulate matter as well as water and energy consumption. I would like to integrate it at Airflow/GreenWall

**Comment by JanineBardey** [comment]
Yes, but I am still trying to find time for it

**Comment by JanineBardey** [comment]
Reopened to continue the work and reference the work in a proposal. The work might take some more time due to other projects running in parallel

**Comment by FelixStege** [comment]
@JanineBardey, what is the status of this? Maybe this is something for the upcoming hackday?

**Comment by JanineBardey** [comment]
@FelixStege 
Future work is planed on this but probably not at the next hackday. 
Please keep the branch open as it is part of a research proposal, which will hopefully help me to further work on this topic.
